tests/unit/store/settings.spec.js

```diff
@@ -111,42 +111,42 @@ describe("store/settings", () => {
         user_ids: array_of_userid,
       },
     ];
-    expect(store.state.settings.customer_account_ids.length).toEqual(0);
+    expect(store.state.settings.customer_account_ids).toHaveLength(0);
     store.commit("settings/set_customer_account_ids", array_of_customerids);
     store.commit("settings/set_customer_index", 0);
     store.commit("settings/set_user_index", 0);
     expect(
       store.state.settings.customer_account_ids[
         store.state.settings.customer_index
       ].uuid
-    ).toEqual("4vqyd62oARXqj9nRUNhtLQ");
+    ).toStrictEqual("4vqyd62oARXqj9nRUNhtLQ");
     expect(
       store.state.settings.customer_account_ids[
         store.state.settings.customer_index
       ].api_key
-    ).toEqual("941532a0-6be1-443a-a9d5-d57bdf180a52");
+    ).toStrictEqual("941532a0-6be1-443a-a9d5-d57bdf180a52");
     expect(
       store.state.settings.customer_account_ids[
         store.state.settings.customer_index
       ].nickname
-    ).toEqual("Customer account -1");
+    ).toStrictEqual("Customer account -1");
     expect(
       store.state.settings.customer_account_ids[
         store.state.settings.customer_index
-      ].user_ids.length
-    ).toEqual(1);
+      ].user_ids
+    ).toHaveLength(1);
     expect(
       store.state.settings.customer_account_ids[
         store.state.settings.customer_index
       ].user_ids[store.state.settings.user_index].uuid
-    ).toEqual("2VSckkBYr2An3dqHEyfRRE");
+    ).toStrictEqual("2VSckkBYr2An3dqHEyfRRE");
     expect(
       store.state.settings.customer_account_ids[
         store.state.settings.customer_index
       ].user_ids[store.state.settings.user_index].nickname
-    ).toEqual("User account -1");
+    ).toStrictEqual("User account -1");
   });
-  test("Store a single customer detail with multiple user ids can be assigned to the customer_ids array", () => {
+  test("When initialized the array of customer_account_ids is empty and size 0, Then commit customer_details with multiple user_ids in customer_account_ids and assert the number of user_ids to match the number of user_ids", () => {
```

Comment:
similar comment as previous test.  "Then" should only contain information about what you are checking.  If you are doing commits, that is "When" (or "Given", depending on how the test works)

Reply:
Would like to retain the present title, its more informative.